tools/transformer/src/atlasTransformations.js

```diff
@@ -67,27 +56,27 @@ module.exports = function runTransformations(openapi) {
     ignoredModelNames
   );
 
-  // Temp workaround for
-  // https://jira.mongodb.org/browse/CLOUDP-166120
-  openapi.components.responses.noBody = {
-    content: {
-      "application/vnd.atlas.2023-01-01+json": {
-        example: "",
-      },
-    },
-    description: "This endpoint does not return a response body",
-  };
-
-  // Temp workaround for CLOUDP-168427
-  if (openapi.components.schemas.Error) {
-    openapi.components.schemas.Error.properties.parameters.items = {};
+  if (openapi.components.schemas.ApiError) {
+    openapi.components.schemas.ApiError.properties.parameters.items = {};
+  }
+
+  if (
+    openapi.components.schemas.ApiAtlastokenFiltersViewManual?.tokenFilters
+      .items
+  ) {
+    openapi.components.schemas.ApiAtlastokenFiltersViewManual.tokenFilters.items =
+      {};
+  }
+  if (openapi.components.schemas.TokenFiltersViewManual?.tokenFilters.items) {
```

Comment:
@swattyT @matt-condon Do we use tokenFilters? Would generalization here makes sense.
There are many objects in that list that will likely to polute our docs and have little value on their own